### PR TITLE
[No Ticket] Install Trivy to scan docker image

### PR DESCRIPTION
**Pull request by zbedo**
This PR introduces: 

- Trivy installation
- Docker image security scan, if there are CRITICAL findings, job will fail.

### CODE REVIEW

**Review by kshakir — APPROVED** [review]
If jenkins is happy I'm happy.

File: .circleci/config.yml [hunk 1]
```diff
@@ -86,6 +86,15 @@ jobs:
...
             if [[ ${CIRCLE_BRANCH} == "dev" ]]; then docker tag ${IMAGE_NAME}:${CIRCLE_BRANCH} ${IMAGE_NAME}:latest ; fi
 
+      - run:
+          name: Install trivy
+          command: |
+            curl -sfL https://raw.githubusercontent.com/aquasecurity/trivy/master/contrib/install.sh | sudo sh -s -- -b /usr/local/bin
+  
+      - run:
+          name: Scan the local image with trivy (light)
+          command: trivy --exit-code 1 --severity CRITICAL --no-progress ${IMAGE_NAME}:latest
```
Comment (kshakir) [inline .circleci/config.yml]: Whoops. I take that back. See the `if` statement above where sometimes the tag isn't `latest`, it's sometimes `${CIRCLE_BRANCH}`.

**Review by kshakir — CHANGES_REQUESTED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (zbedo) [reply .circleci/config.yml]: Thank you, nice catch!

**Review by kshakir — APPROVED** [review]

**Review by rsasch — APPROVED** [review]

**Comment by kshakir** [comment]
@zbedo Did you want to wait for @adrazhi to review, or shall we merge this now?

**Review by adrazhi — APPROVED** [review]

**Comment by zbedo** [comment]
@kshakir I think we can merge it now.

**Pull request merged**